### PR TITLE
Integrate RNTester (UIExplorer)

### PR DESCRIPTION
**Pull request by saschatimme**
As discussed in #16.

I renamed it from UIExplorer to RNTester to have the same naming as the react-native repo.

Could you verify that the setup works for you @wokalski

### CODE REVIEW

**Comment by wokalski** [comment]
I cannot build it sadly :( 
```
error: bundling: UnableToResolveError: Unable to resolve module `bs-platform/lib/js/curry.js` from `/Users/wczekalski/Documents/Work/Projects/Discover/bs-react-native/lib/js/RNTester/src/rNTesterApp.js`: Module does not exist in the module map or in these directories:
  /Users/wczekalski/Documents/Work/Projects/Discover/bs-react-native/node_modules/bs-platform/lib/js
,   /Users/wczekalski/Documents/Work/Projects/Discover/node_modules/bs-platform/lib/js
```

**Comment by wokalski** [comment]
Also, we should exclude the tester app from the npm package with `.npmignore`.

**Comment by saschatimme** [comment]
I tweaked everything a little bit and updated the instructions, so that it should now work.
Will do the rest later in the day

**Comment by wokalski** [comment]
cool. I'll test it later, too!

**Comment by wokalski** [comment]
The two commits I pushed mess up styles a bit - they change styles to floating point numbers.

File: RNTester/src/RNTesterApp.re [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        "header":
+          style [
+            height 60,
+            borderBottomWidth StyleSheet.hairlineWidth,
+            borderBottomColor "#96969A",
+            backgroundColor "#F5F5F6",
+            flexDirection `row,
+            paddingTop 20
+          ],
+        "headerLeft": style [],
```
Comment (saschatimme) [inline RNTester/src/RNTesterApp.re]: Just copied from the original thing and didn't give it much thought :D

File: RNTester/index.android.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    margin: 10,
+  },
+  instructions: {
+    textAlign: 'center',
+    color: '#333333',
+    marginBottom: 5,
+  },
+});
+
+AppRegistry.registerComponent('RNTester', () => RNTester);
```
Comment (saschatimme) [inline RNTester/index.android.js]: good catch

File: RNTester/src/RNTesterBlock.re [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          ],
+        "titleText": style [fontSize 14, fontWeight `_500],
+        "descriptionText": style [fontSize 14],
+        "disclosure": style [position `absolute, top 0, right 0, padding 10],
+        "disclosureIcon": style [width 12, height 8],
+        "children": style [margin 10]
+      }
+    );
+
+module RNTesterBlock = {
```
Comment (saschatimme) [inline RNTester/src/RNTesterBlock.re]: now it's a module^^

Comment (saschatimme) [inline RNTester/src/RNTesterBlock.re]: but sure

**Comment by saschatimme** [comment]
Changed the `index.android.js`. If you can confirm that you can setup everything we are ready to go :)

**Comment by saschatimme** [comment]
Forgot the npm `package.json` stuff, will give it a look

**Comment by wokalski** [comment]
👏

**Pull request merged**